include/cvc5/cvc5.h

```diff
@@ -1941,6 +1941,13 @@ class CVC5_EXPORT Term
   std::shared_ptr<internal::Node> d_node;
 };
 
+/**
+ * Get the number of indices for a skolem id.
+ * @param id The skolem id.
+ * @return The number of indices for the skolem id.
+ */
+CVC5_EXPORT size_t getNumIndicesForSkolemId(SkolemId id);
```

Comment:
I think the main implementation of this method should live internally in `SkolemManager`.

Moreover, I am not sure if we should expose this method in the API, or if there is a way for the parser to access it via a backdoor.

Reply:
Couldn't figure out a good way for the parser to access this without exposing it in the api